### DISCUSSION TITLE
Assistance in training the model

### DISCUSSION THREAD

**Discussion by trcarthew** [THUDM/CogVLM#344] [General]
We are a startup that are looking at making use of the CogVLM model in a project we are working. We need to achieve a high quality of structured data output from the model, so need to perform some fine tuning to achieve the result.

We are looking to contract someone to help setup the infrastructure and pipeline for this. Does anyone know anyone who might be able to help?

**Comment by mohit-217** [comment]
@trcarthew Hello ,could you please elaborate more.
